From 73b772bda5cb2ddaaba295468cc69ab71bba9ca0 Mon Sep 17 00:00:00 2001
From: Mandy Lavi <mandy.lavi@freescale.com>
Date: Mon, 24 Feb 2014 17:04:47 +0200
Subject: [PATCH 580/987] fmd: correct compilation warning in
 lnxwrp_sysfs_fm_port.c

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Change-Id: I1d0661f217d96c530ad03e797be85edf2c8a3b1e
Signed-off-by: Mandy Lavi <mandy.lavi@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/9077
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Nir Erez <nir.erez@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
(cherry picked from commit 431aea6cc98e1c0350214dc3bdb2a32269307211)
Change-Id: I1d0661f217d96c530ad03e797be85edf2c8a3b1e
Reviewed-on: http://git.am.freescale.net:8181/9431
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
---
 drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm_port.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm_port.c b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm_port.c
index 4b743da..73db24b 100644
--- a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm_port.c
+++ b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm_port.c
@@ -152,7 +152,7 @@ static ssize_t show_fm_port_stats(struct device *dev,
 	}
 
 	local_irq_save(flags);
-	n = snprintf(buf, PAGE_SIZE, "\t%s counter: %lu\n",
+	n = snprintf(buf, PAGE_SIZE, "\t%s counter: %u\n",
 		p_LnxWrpFmPortDev->name,
 		FM_PORT_GetCounter(p_LnxWrpFmPortDev->h_Dev,
 					(e_FmPortCounters) counter));
-- 
1.9.1

